src/ucp/core/ucp_request.inl

```diff
@@ -160,15 +159,16 @@ static UCS_F_ALWAYS_INLINE void
 ucp_request_put(ucp_request_t *req)
 {
     ucs_trace_req("put request %p", req);
-    ucs_assert(!(req->flags & UCP_REQUEST_FLAG_IN_PTR_MAP));
+    ucs_assert(!(req->flags & UCP_REQUEST_FLAG_LOCAL_ID));
     UCS_PROFILE_REQUEST_FREE(req);
     ucs_mpool_put_inline(req);
 }
 
 static UCS_F_ALWAYS_INLINE void
 ucp_request_complete_send(ucp_request_t *req, ucs_status_t status)
 {
-    ucs_trace_req("completing send request %p (%p) "UCP_REQUEST_FLAGS_FMT" %s",
+    ucs_trace_req("completing send request %p (%p) " UCP_REQUEST_FLAGS_FMT
```

Comment:
unrelated?

Reply:
unfortunately, it is related.
`src/ucp/core/ucp_request.inl` can't be compiled with C++ - to include in gtest